etc/behat/services/contexts/ui.xml

```diff
@@ -40,6 +40,7 @@
             <argument type="service" id="sylius.behat.page.admin.locale.create" />
             <argument type="service" id="sylius.behat.page.admin.locale.index" />
             <argument type="service" id="sylius.behat.service.accessor.notification_accessor" />
+            <argument type="service" id="sylius.behat.page.admin.locale.update" />
```

Comment:
Yeah i felt uncomfortable when i was adding this but we decided to put parameters in alphabetical order.